src/lib/date.ts

```diff
@@ -254,15 +254,27 @@ export function formattedOpeningHours(startAt, endAt, timezone) {
  * Ends Apr 3 at 12:30pm
  * Ended Apr 3 2017
  */
-export function formattedStartDateTime(startAt, endAt, timezone) {
-  const thisMoment = moment()
+export function formattedStartDateTime(startAt, endAt, liveStartAt, timezone) {
```

Comment:
this method seems pretty generic, adding `liveStartAt` makes it very "auction" related should we make a separate method that possibly uses this method and extends it with `liveStartAt`?

Reply:
If liveStartAt is null it should resolve to the same strings as it did before. I also checked and confirmed that this is only used in emission for auction times.

Reply:
cool, thanks for confirming. I would still rename it to `auctionFormattedDateTime` to reflect the auction specific logic in there since this method is defined in pretty generic area `src/lib/date.ts`. what do you think?

Reply:
I discussed that with Sarah, but because prod still depends on the other name we thought it was more trouble than its worth. Also it still works for non-auctions.

Reply:
👍